src/WebJobs.Script/Utility.cs

```diff
@@ -159,5 +160,16 @@ public static string ToLowerFirstCharacter(string input)
 
             return input;
         }
+
+        public static string RemoveUtf8ByteOrderMark(string input)
```

Comment:
Perhaps you should remove TestHelpers.RemoveByteOrderMark and use this everywhere instead - fives it more testing